internal/http/services/owncloud/ocdav/ocdav.go

```diff
@@ -232,7 +234,45 @@ func (s *svc) Handler() http.Handler {
 				http.Redirect(w, r, rURL, http.StatusMovedPermanently)
 				return
 			}
+		case "ocm-provider":
+			// Reverse proxy of /ocm/ocm-provider, assuming it is exposed in localhost. When not configured, returns 404.
+			// TODO(lopresti) this as well as other endpoints here should go to some common place for
+			// routes served from the root, including the reverseproxy service (which if configured
+			// together with ocdav would just override it!).
+			proto := "https"
+			if s.c.Insecure {
+				proto = "http"
+			}
+			ocmEP := fmt.Sprintf("%s://localhost/ocm/ocm-provider", proto)
```

Comment:
I think this will use the default ports, but this is not always true. For example at CERN this is exposed in the port 9143...

Reply:
yes, too many assumptions with this approach - we go for a separate microservice